src/formatting.jl

```diff
@@ -208,9 +208,9 @@ end
 function format_published_in(entry; include_date=true)
     str = ""
     if entry.type == "article"
-        str *= entry.in.journal
+        str *= replace(entry.in.journal, " " => " ")  # non-breaking space
```

Comment:
These two strings both contain only U+0020 (the regular space character) for me. I'd use the `\uxxxx` mode if a specific replacement is needed:

```julia-repl
julia> "\u01" |> first
'\x01': ASCII/Unicode U+0001 (category Cc: Other, control)
```

Reply:
Agreed: it's not a good idea to have invisible unicode in the code. I'll fix it. I was being lazy, and I have to figure out the code point for a non-breaking space. 

But when you say "These two strings both contain only the regular space character", that seems to be a bug in the Github UI! In the committed code, the second space is non-breaking, but I can confirm that if I copy and paste the above from the web UI into my text editor, both strings are regular spaces.